### PR TITLE
Consolidate scattered max peer connection values into single declaration.

### PR DESCRIPTION
**Pull request by ghost**
Currently, there are several integer literals of value 125 used to represent the maximum number of peer connections to maintain. This commit replaces those literals with a single constant defined in net.h.

### CODE REVIEW

**Comment by jonasschnelli** [comment]
utACK.

**Comment by NanoAkron** [comment]
I forget by whom, but wasn't there a previous recent effort to consolidate all constants into a single consensus header?

**Comment by sipa** [comment]
@NanoAkron This is just for human consumption, definitely not a consensus-critical constant.

**Comment by laanwj** [comment]
Looks good to me.
utACK

File: src/net.h [hunk 1]
```diff
@@ -56,6 +56,8 @@ static const bool DEFAULT_UPNP = false;
 #endif
 /** The maximum number of entries in mapAskFor */
 static const size_t MAPASKFOR_MAX_SZ = MAX_INV_SZ;
+/** The maximum number of peer connections to maintain. */
+static const unsigned int MAX_PEER_CONNECTIONS = 125;
```
Comment (laanwj) [inline src/net.h]: A more consistent (and correct) name would be `DEFAULT_MAX_PEER_CONNECTIONS`

**Comment by laanwj** [comment]
Similar to #6349 (but affects a different value)

File: src/net.h (hunk 1 above)
Comment (Diapolo) [inline src/net.h]: Agreed

**Comment by fanquake** [comment]
utACK

**Comment by sipa** [comment]
utACK

**Pull request merged**